### ISSUE TITLE
Sharing trained weight

### ISSUE THREAD

**Issue by jeammimi** [maxhodak/keras-molecules#10]
Hello, I am very interested in testing your model.
Do you think you could share the trained model file (model.h5 file)
Thank you

**Comment by maxhodak** [comment]
I need to train a new model given recent changes to model.py but I'll include a trained model (via git-lfs, ~100mb) once it's done.

**Comment by jeammimi** [comment]
Ok, thanks.
I tried to train the model on some molecules extracted from a patent
database, and my accuracy is kind of low: 0.72 There is about 1millions of
molecules. What kind of accuracy do you get?

Le 6 nov. 2016 3:03 PM, "Max Hodak" notifications@github.com a écrit :

I need to train a new model given recent changes to model.py but I'll
include a trained model (via git-lfs, ~100mb) once it's done.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/maxhodak/keras-molecules/issues/10#issuecomment-258682813,
or mute the thread

**Comment by maxhodak** [comment]
The pretrained model is now included in `data/model_500k.h5`. It works with `data/smiles_500k.h5` (you must feed that data file through the preprocessing script first).

I was seeing an accuracy of about 0.98 when I stopped training that model (60 epochs, ~12 hours on a GTX 1080).

**Comment by michaelosthege** [comment]
could someone post the charset that the model was trained for?
I'm having difficulties to reproduce (mostly due to Python version..). In my re-implementation I'm sorting the charset and save it to a .json file alongside the trained model.
